src/traces/bar/layout_attributes.js

```diff
@@ -51,5 +51,14 @@ module.exports = {
             'Sets the gap (in plot fraction) between bars of',
             'the same location coordinate.'
         ].join(' ')
-    }
+    },
+    barcornerradius: {
+        valType: 'any',
+        dflt: 0,
```

Comment:
I think we should remove this `dflt: 0` line; so that we have undefined `cornerradius` defaults for stacked bars.